cmd/core-common-config-bootstrapper/res/configuration.yaml

```diff
@@ -0,0 +1,135 @@
+all-services:
+  Writable:
+    LogLevel: "INFO"
+    
+    InsecureSecrets:
+      DB:
+        path: "redisdb"
+        Secrets:
+          username: ""
+          password: ""
+    
+    Telemetry:
+      Interval: "30s"
+      PublishTopicPrefix : "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
+      Metrics:
+        # Common Security Service Metrics
+        SecuritySecretsRequested: false
+        SecuritySecretsStored: false
+        SecurityConsulTokensRequested: false
+        SecurityConsulTokenDuration: false
+      Tags: # Contains the service level tags to be attached to all the service's metrics
+      #  Gateway: "my-iot-gateway" # Tag must be added here or via Consul Env Override can only change existing value, not added new ones.
+      
+  Service:
+    HealthCheckInterval: "10s"
+    Host: "localhost"
+    ServerBindAddr: "" # Leave blank so default to Host value unless different value is needed.
+    MaxResultCount: 1024
+    MaxRequestSize: 0 # Not currently used. Defines the maximum size of http request body in bytes
+    RequestTimeout: "5s"
+    CORSConfiguration:
+      EnableCORS: false
+      CORSAllowCredentials: false
+      CORSAllowedOrigin: "https://localhost"
+      CORSAllowedMethods: "GET, POST, PUT, PATCH, DELETE"
+      CORSAllowedHeaders: "Authorization, Accept, Accept-Language, Content-Language, Content-Type, X-Correlation-ID"
+      CORSExposeHeaders: "Cache-Control, Content-Language, Content-Length, Content-Type, Expires, Last-Modified, Pragma, X-Correlation-ID"
+      CORSMaxAge: 3600
+
+  Registry:
+    Host: "localhost"
+    Port: 8500
+    Type: "consul"
+  
+  Databases:
+    Primary:
+      Host: "localhost"
+      Port: 6379
+      Timeout: 5000
+      Type: "redisdb"
+  
+  MessageBus:
+    Protocol: "redis"
+    Host: "localhost"
+    Port: 6379
+    Type: "redis"
+    AuthMode: "usernamepassword"  # required for redis MessageBus (secure or insecure).
+    SecretName: "redisdb"
+    Topics:
+      PublishTopicPrefix: "edgex/events/core" # /<device-profile-name>/<device-name> will be added to this Publish Topic prefix
+      SubscribeTopic: "edgex/events/device/#"  # required for subscribing to Events from MessageBus
+    Optional:
+      # Default MQTT Specific options that need to be here to enable environment variable overrides of them
+      ClientId: "core-data"
```

Comment:
should we use different id here? or this is just a placeholder

Reply:
Probably should just remove it since local private config will have to override it.,

Reply:
removed ClientId